### ISSUE TITLE
Question about the number of keypoints in Model.

### ISSUE THREAD

**Issue by puhuajiang** [open-mmlab/mmskeleton#365]
I used the "configs/utils/build_dataset_example.yaml" to generate my datasets to test the model. It successfully generated the json file which indicated that the number of keypoints is 17. So when I used the pth in "checkpoints/st_gcn.kinetics-6fa43f73.pth" to test the datasets. An error occurred :

size mismatch for edge_importance.8: copying a param with shape torch.Size([3, 18, 18]) from checkpoint, the shape in current model is torch.Size([3, 17, 17]). 

SO, if I want to use a model which the number of keypoints is  17, I must train it for myself ?

**Comment by zhou07** [comment]
hello, I meet the same question about the number of keypoints as you, do you solve the problem now?

**Comment by puhuajiang** [comment]
> hello, I meet the same question about the number of keypoints as you, do you solve the problem now?

yes, I have written a simple python script to change the COCO skeleton to Openpose skeleton. 

def HR2openpose(HR_points):
    openpose_points = np.zeros((18,3))
    openpose_points[0] = HR_points[0]
    openpose_points[1] = (HR_points[5] + HR_points[6]) /2
    openpose_points[2] = HR_points[5]
    openpose_points[3] = HR_points[7]
    openpose_points[4] = HR_points[9]
    openpose_points[5] = HR_points[6]
    openpose_points[6] = HR_points[8]
    openpose_points[7] = HR_points[10]
    openpose_points[8] = HR_points[11]
    openpose_points[9] = HR_points[13]
    openpose_points[10] = HR_points[15]
    openpose_points[11] = HR_points[12]
    openpose_points[12] = HR_points[14]
    openpose_points[13] = HR_points[16]
    openpose_points[14] = HR_points[1]
    openpose_points[15] = HR_points[2]
    openpose_points[16] = HR_points[3]
    openpose_points[17] = HR_points[4]
    return openpose_points

I hope this simple function can help you .

**Comment by zhou07** [comment]
Thank you!

**Comment by zhou07** [comment]
Thank you very much for this function, it is very useful！

**Comment by Yliqiang** [comment]
34/5000 
I think you only need to modify the layout in train.yaml to 'coco'.
